config/sync/field.field.node.stanford_page.su_page_image.yml

```diff
@@ -0,0 +1,29 @@
+uuid: f7fbb866-497b-446c-8d9d-dc3cf62dcbee
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.su_page_image
+    - media.type.image
+    - node.type.stanford_page
+id: node.stanford_page.su_page_image
+field_name: su_page_image
+entity_type: node
+bundle: stanford_page
+label: 'Page Image'
+description: 'Maximum one image. The <i>"page image"</i> will only appear as a thumbnail image on Teaser and List page display. Use the Banner paragraph to add a Featured Media to a page.'
```

Comment:
Should we also mention it'll be used at the social share image and put some recommendations on size/ratio?